src/main/java/org/openrewrite/kotlin/internal/KotlinSource.java

```diff
@@ -56,6 +56,10 @@ private Map<Integer, ASTNode> map(@Nullable PsiFile psiFile) {
 
         String source = input.getSource(new InMemoryExecutionContext()).readFully();
         Set<PsiElement> visited = Collections.newSetFromMap(new IdentityHashMap<>());
+
+        System.out.println(PsiTreePrinter.printPsiTree(psiFile));
```

Comment:
We shouldn't print anything by default because this will appear in the build tools and CLI outputs.